extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -319,7 +320,7 @@ private function getFeaturedSection( $content ) {
 	 */
 	private function setSectionItemsResponse( $sectionName, $ret ) {
 		foreach ( $ret as &$value ) {
-			list( $image_id, $image_url ) = CuratedContentSpecialController::findImageIfNotSet(
+			list( $image_id, $image_url ) = CuratedContentHelper::findImageIfNotSet(
```

Comment:
Not a fan of Helper functions as they tend to become PutSomeRandomMethodsClass.php
'Memory is cheap' can we divide it into some logical classes?
CuratedContentImage
CuratedContentUrl

maybe even those can live somewhere else and be helpful to other teams?